app/components/geoblacklight/header_icons_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Geoblacklight
+  class HeaderIconsComponent < ViewComponent::Base
```

Comment:
could we just make this `Earthworks::HeaderIconsComponent` too? I think it's easier to know when things are changed if they get namespaced that way

Reply:
Yeah I thought about that but then I think we need to pull out a bunch of components from geoblacklight and override those as well.

Reply:
unless there's more places that aren't in that search, I think that is OK (and in fact good). the things in that search are:
- the component itself, which we are already changing
- a partial, which we would change to use the new component
- a test, which is not in earthworks
- a helper function, which we would change to use the new component
- the component that references this component, which is already changed in this PR

that seems like a pretty easy lift to me! just two places that aren't already in the PR, and neither is a component.

Reply:
I would be more inclined to change the way geoblacklight uses the component. Is there a reason we wouldn't want to update the config to set the headericon class in the controller?

Reply:
that is another path, for sure. from what i've seen that is mostly used for "big" stuff like the document component, sidebar, etc. but if that seems easier to you, go for it

Reply:
@thatbudakguy actually the helper I am pretty sure we don't want using our custom headericonscomponent. And the https://github.com/geoblacklight/geoblacklight/blob/9ce0dccdc2336dad520dfc0578743b20d421b0f6/app/views/catalog/_show_header_default.html.erb#L3 actually needs to get removed and fixed in geoblacklight (the document component missed this).

I am updating this PR.